releasenotes/notes/add-aws-api-gateway-dabfeee3decdfd89.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Add preview support to create inferred AWS API Gateway spans when certain AWS Gateway headers are passed into the request. 
+    Requires ``DD_TRACE_INFERRED_PROXY_SERVICES_ENABLED=true`` to enable. For enablement instructions, `contact support. https://docs.datadoghq.com/help/`_
```

```suggestion
    Requires ``DD_TRACE_INFERRED_PROXY_SERVICES_ENABLED=true`` to enable. For enablement instructions, contact support. https://docs.datadoghq.com/help/
```
Do we want tick marks around the contact support section? Maybe I don't know what the formatting does.

---

Do we use sphinx for the formatting? If so, then this should create a link: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html . (But I have no idea how to preview this).

---

Oh actually I found that LLM recently also added a link to their release notes then adjusted this branch to do that too: https://github.com/DataDog/dd-trace-py/blob/122caa61c278a75100d7c23fc601588cc0c60308/releasenotes/notes/ragas-integration-a81b696757c0e7a5.yaml#L12C44-L12C149 .

I'll mark this as resolved since I think it's important to link to the contact support link for this.